CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### Added
+- Created create/select-template page and test
```

Comment:
@fraserclark, could you add the ticket # this change is related to in the CHANGELOG.md file? I think it will be helpful being able to see that info here.

And also, if you could remember to link the ticket in the PR description as well, so we can easily find this related PR when we are looking at the ticket.